conf/include/bblayers/sota.inc

```diff
@@ -1,4 +1,6 @@
-
+BBLAYERS += "${METADIR}/poky/meta"
+BBLAYERS += "${METADIR}/poky/meta-poky"
+BBLAYERS += "${METADIR}/poky/meta-yocto-bsp"
```

Comment:
This seems to cause my new bblayers.conf to have duplicated entries for each of these three layers. Do we need this?

Reply:
Um... no, I was sure I've deleted that.

Reply:
Just fixed these, thanks. There are some minor fixes coming for RPi3 and maybe Minnowboard, but I'm not sure if it's worth adding them to this PR.